services/blockchain-connector/shared/sdk/genesisBlock.js

```diff
@@ -83,9 +84,68 @@ const getGenesisConfig = async () => {
 	}
 };
 
+const getGenesisAssets = async (params = {}) => {
+	const rawGenesisBlock = await getGenesisBlock(true);
+	const genesisBlock = await formatBlock(rawGenesisBlock);
+
+	const assetModule = genesisBlock.assets.find(
+		genesisAsset => genesisAsset.module === params.module,
+	);
+
+	if (params.subStore) {
+		const assetModuleSubStoreKey = Object.keys(assetModule.data).find(
+			subStoreKey => subStoreKey === params.subStore,
+		);
+
+		return assetModuleSubStoreKey
+			? [{
+				...assetModule,
+				data: {
+					[assetModuleSubStoreKey]: assetModule.data[assetModuleSubStoreKey],
+				},
+			}]
+			: [];
+	}
+
+	if (params.module) {
+		return assetModule ? [assetModule] : [];
+	}
+
+	return genesisBlock.assets;
```

Comment:
This will result in inconsistency in response structure. Please always return the response in the following format:
```
{
 "subStore1": [],
 "subStore2": [],
}
```

Reply:
As per previous discussion this method was supposed to return an array of asset filtered by module and subStore key. seems to be returning that structure for empty param, module, module and subStore filtering. Will rename the endpoint to `getAssetByModule` and return the suggested structure.